scripts/test_env.py

```diff
@@ -148,8 +150,8 @@ def run(self):
         logging.info(f"Running peer {self.name}...")
 
         # FD never gets closed
-        stdout_file = open(self.peer_dir / ".stdout", "w")
-        stderr_file = open(self.peer_dir / ".stderr", "w")
+        stdout_file = open(self.peer_dir / "log.stdout", "w")
+        stderr_file = open(self.peer_dir / "log.stderr", "w")
```

Comment:
is it possible to put logs in the single file?

Reply:
Yeah, that's probably more convenient.